tests/marathon_one_app_two_health_checks.json

```diff
@@ -117,7 +117,7 @@
         "healthChecks": [
             {
                 "path": "/this/is/a/nondefault/path",
-                "protocol": "HTTP",
```

Comment:
Should we test both old and new health check protocols (i.e., retain old ones and add new tests for the new ones)?

Reply:
Converted one test to use the new MESOS_HTTP health-check type, there are still several other tests that still use the legacy HTTP type.